test_conformance/geometrics/test_geometrics.cpp

```diff
@@ -265,10 +268,11 @@ int test_geom_cross(cl_device_id deviceID, cl_context context, cl_command_queue
 
     if(!is_extension_available(deviceID, "cl_khr_fp64")) {
         log_info("Extension cl_khr_fp64 not supported; skipping double tests.\n");
-        return 0;
+        return error;
```

Comment:
you should use enum test_status . In this case TEST_SKIP ?

Reply:
are you referring to the log_info macro?
I don't think this relates to my change, does it?

Reply:
I mean enum from testHarness.h -> typedef enum test_status . In case extension is not supported we TEST_SKIP error code should be returned